test/unit/controllers/category-controller-spec.js

```diff
@@ -40,7 +40,7 @@
       });
 
       it('providers should exist', function () {
-        expect(ctrl.providers).to.exist;
+        expect(ctrl.providers).to.exist; //jshint ignore:line
```

Comment:
da errores ahi al momento de correr los test revise categories-controller-spec (expect(ctrl.categories).to.not.be.null; //jshint ignore:line) y a ese expect le habian puesto essa linea por eso la puse aca tambien para que no haga caer los tests por ese jshint